apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt_egress.erl

```diff
@@ -22,13 +22,16 @@
 
 -export([
     config/1,
-    send/3,
-    send_async/4
+    send/4,
+    send_async/5
 ]).
 
 -type message() :: emqx_types:message() | map().
 -type callback() :: {function(), [_Arg]} | {module(), atom(), [_Arg]}.
 -type remote_message() :: #mqtt_msg{}.
+-type trace_rendered_func() :: {
```

Comment:
export type from emqx_trace.erl

Reply:
Will do and will also extract common functionality as you suggested in the comment below.